### ISSUE TITLE
Reporting a clip should act as a No vote

### ISSUE THREAD

**Issue by dabinat** [mozilla/voice-web#2111]
It appears that when you report a clip during validation, it behaves as a skip. This means that a reported clip can keep appearing again and again and it took me a few tries of reporting the same clip over and over to realize this behavior.

As it stands right now, I need to report a clip, then wait for it to appear again in order to reject it (and of course manage to remember that I’ve heard it before and previously reported it).

It seems more logical that reporting a clip should act as a rejection. You wouldn’t report a clip that’s acceptable and I can’t think of a reason why you’d need it to appear again.

**Comment by Gregoor** [comment]
Good point, I'll ask the rest of the team for comments.

**Comment by kdavis-mozilla** [comment]
Subtle points might occur with "Grammatical / spelling error" and "Other".

For example, an American English speaker might mark the word "summarise" as a spelling error while a British English speaker might mark "summarize" as a spelling error. Same language two different spellings for the same word.

**Comment by dabinat** [comment]
@kdavis-mozilla Right, but this would just be for that particular user. It wouldn't prevent anyone else from seeing the clip. It would be the equivalent of the user pressing the No button.

**Comment by kdavis-mozilla** [comment]
@dabinat But then all it takes is 2 American English speakers or 2 UK English speakers to remove a clip which is actually valid.

**Comment by dabinat** [comment]
@kdavis-mozilla Sure, but that could happen regardless of whether or not they report the clips. They could each manually vote no. So this doesn't really change that.

My point is that I can't see a situation where you would report a clip and then vote yes for it, so this saves the reporting user from having to manually reject the clip the next time it comes up.

**Comment by kdavis-mozilla** [comment]
@dabinat My point is that one language may have two or more definitions of "Grammatical/spelling error", say through different accents; so having a "Grammatical/spelling error" count as a "manually vote no" in this case will remove such examples from the data set thus devaluing the use of the data set for accent fine tuning.

In particular it would be exactly these type of examples you would want in a data set used for accent fine tuning, and this would remove them.

**Comment by dabinat** [comment]
@kdavis-mozilla I understand what you're saying, but that circumstance can still occur regardless of the reporting feature, and is probably something that has been occurring since day one of Common Voice. The crowd-sourcing model will always have some degree of error - some bad clips will make it through, some good clips won't, but ideally those are small margins that balance each other out and the vast majority of clips end up in the correct place.

I don't think this feature makes any difference to that. Adding an automatic No vote saves me from manually voting No later on. There isn't a situation in which I would report a clip and then vote yes for it. I don't see why a report should be considered different from a No vote, when a No vote is in fact easier and quicker to achieve.

What you've brought up is a different issue - the quality of validation. That's certainly worth discussing, but I don't think this issue makes any difference to that. This issue is about **how** the user rejects a clip, but your question is more about **why** they are choosing to reject it in the first place. That could be better addressed by educating users or requiring more votes per clip. 

(Also, as a side note, I am originally British and there's enough US culture in the UK that most Brits should be familiar with how Americans spell / pronounce common words. I don't know how many Brits would consider those "wrong" to the extend that the sentence should be reported as incorrectly spelled.)

But if there is widespread opposition to this, two alternatives would be:

1. Don't skip the clip - require the user to manually make a validation decision after reporting it.
2. Don't show reported clips again to the user who reported them.

**Comment by kdavis-mozilla** [comment]
@dabinat This...

> ...that circumstance can still occur regardless of the reporting feature...

is false. Per accent validation would fix this.

Also on this...

> Adding an automatic No vote saves me from manually voting No later on

Thee point is not about a single person. The point is about the utility of the entire data set. This will decrease that utility. This much is obvious.

The question is: Can we sharpen "Grammatical/spelling error" to avoid this problem.

**Comment by kdavis-mozilla** [comment]
@dabinat I don't think we should close it. I think we just need to figure out how to sharpen "Grammatical/spelling error" to avoid the problem I was talking about.

**Comment by ftyers** [comment]
Closing this for now.
